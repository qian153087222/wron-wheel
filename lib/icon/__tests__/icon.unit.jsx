import renderer from 'react-test-renderer';
import React from 'react';
import Icon from '../icon' 

describe('icon',()=>{
    it('测试ui',()=>{
        const json = renderer.create(<Icon name='qq'/>).toJSON();
        //在当前目录生成一个快照，
        expect(json).toMatchSnapshot();
    })
})
