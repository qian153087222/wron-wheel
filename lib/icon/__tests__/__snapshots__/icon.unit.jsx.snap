// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon 测试Iconui 1`] = `
<svg
  className="wron-icon"
>
  <use
    xlinkHref="#qq"
  />
</svg>
`;
